install_jupyter_ros.sh

```diff
@@ -9,21 +9,21 @@ sudo apt install -y python3 python3-venv python3-dev git libffi-dev
 # Due to a build error on numpy we need to install numpy and
 # pandas globally and us it in the virtual environment
 # TODO: check if (micro/mino)mamba/conda could fix this
-cd /home/mirte || exit
+cd /home/mirte || exit 1
 sudo apt install -y python3-numpy python3-pandas
 python3 -m venv jupyter --system-site-packages
 source /home/mirte/jupyter/bin/activate
 
 # install jupyros
 sudo apt install -y libjpeg-dev zlib1g-dev
 pip3 install wheel
-pip3 install markupsafe==2.0.1 pyzmq==24 zipp==3.1.0
-pip3 install ipython==8.7.0 ipykernel==6.17.1 ipywidgets==7.7.2 \
+pip3 install markupsafe==2.0.1 pyzmq==24 zipp==3.1.0 \
+	ipython==8.7.0 ipykernel==6.17.1 ipywidgets==7.7.2 \
 	jupyter-client==7.4.8 jupyter-core==5.1.0 \
 	nbclient==0.7.2 nbconvert==7.2.6 nbformat==5.7.0 \
-	qtconsole==5.4.0 traitlets==5.6.0
-pip3 install notebook==6.5.2 bqplot==0.12.18 pyyaml
-pip3 install --pre jupyros==0.7.0a0
+	qtconsole==5.4.0 traitlets==5.6.0 \
+	notebook==6.5.2 bqplot==0.12.18 pyyaml \
+	jupyros==0.7.0
```

Comment:
I remember that I had to really split this into certain order (with jupyros the latest) for some reason. Could that be the problem you faced with the OrangePoZero1?

Reply:
Ah, then I'll revert this and I can check it with the opi01

Reply:
If that does not fix the opi1 issue, you can leave it this way. This way is cleaner ;)

Reply:
It is not fixed, still issues